src/search.coffee

```diff
@@ -50,12 +50,13 @@ class Search
       options.mkt = options.market if option.market in markets
       delete options.market
 
-    options.q = @quote options.q
+    options.q = @sanitizeQuery options.q
     options
 
-  quote: (str) ->
-    str = str.replace '"', '\"' # Escape existing quotes.
-    str.replace /^|$/g, '"'     # Quote entire phrase.
+  # Normalize whitespace and then quote phrases.
+  sanitizeQuery: (query) ->
```

Comment:
It'd be good to have tests for this.  That way it'd be easier to understand what the regex does.